### PR TITLE
Add some more people to follow

### PR DESCRIPTION
**Pull request by mxstbr**
Prompted by this thread: https://twitter.com/natkuTala/status/874255873262190592

### CODE REVIEW

File: public/assets/js/core.js [hunk 1]
```diff
@@ -32,7 +32,16 @@ var App = (function (window, document, $) {
...
+            'sarasoueidan',
+            'rmurphey',
+            'bitchwhocodes',
+            'noopkat',
+            'sarah_edo',
+            'ohhoe',
+            'jennschiffer',
+            'kosamari',
+            'cassidoo'
         ],
```
Comment (kentcdodds) [inline public/assets/js/core.js]: Could we add randomization here so it's not a bunch of men, then a bunch of women?

Comment (mxstbr) [inline public/assets/js/core.js]: Yeah I agree, happy to add that in a follow-up PR!

Comment (mxstbr) [inline public/assets/js/core.js]: PR open for shuffling! #308 (I hope git can resolve those merges automatically...)

**Comment by montogeek** [comment]
getify deactivated/suspend his account, Should it be removed?

**Comment by mxstbr** [comment]
And @guillermo should probably be @rauchg (?)—I guess another follow-up PR would be appreciated!

**Comment by gnuwilliam** [comment]
Hey @mxstbr, thanks for the updated list! And my apologies to the female JS community for this huge mistake. Please spread the word and bring girls to be contributors of this project! I'll be more than happy to assist in any way.

**Pull request merged**